docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -394,7 +394,9 @@ dependent on tax forms to have a guardian. This will
 improve tracking for names, and dependent status on tax forms. 
 
 This person will be listed as ["Guardian"]. By design, most will be 
-parents, but some may be a grandparent or other relative. 
+parents, but some may be a grandparent or other relative. This will 
```

Comment:
IMO, describing the state table shouldn't be necessary if we clearly outline how this works. While I'm pretty sure in this case it will be implemented the way you describe here, it could also be implemented the opposite way (each simulant keeps track of who they are the guardian **of**) and that would also work.

Reply:
I think the main thing this section needs is to consistently use the term "guardian" instead of "parent/guardian" -- because "parent" is already a concept in the simulation and we do not want them to be confused with each other.

Also, I would specify here at the top the restriction that each simulant will have a maximum of 2 guardians.

In the rules below, to make it more implementation-agnostic, we can be a bit more explicit, something like saying "assign the reference person to be this simulant's guardian" instead of "assign reference person." Or, we can add some introductory text that explains to interpret the rules this way.

A nit I didn't notice until now: "child"/"parent" are also used in the assignment rules below. This is particularly confusing on [line 421](https://github.com/ihmeuw/vivarium_research/pull/1041/files#diff-45a09ee257e7b14d6545111feb3c2f747ef32aa90038f0dbc028e55d2ff08f0dR421), where I think you are referring to the relationship attribute, not the tracked parent attribute (which of course is not set at initialization).

Reply:
i agree with zeb --- we would like to avoid asking you guys to specifically delineate how the state table (or other within-sim machinery) is explicitly implemented, and instead have the docs outline the concepts you want to exist in the sim / maybe the needs they meet

i think sometimes those lines blur a bit, sorry about that!

i also agree disambiguation would be helpful here --- i think there are 3ish concepts that are getting conflated here:
1. a relationship within the sim we are calling "guardian". currently, the only usage i think you guys have for this is to decide who claims whom in tax returns (will it have other purposes?)
2. a relationship within the sim called "parent". currently, this is a link between the mother of a sim who was born during the simulation, and that child
3. a value in the "relationship to head of household" column called "parent"

it would be really helpful if, each time one of these was referenced, it was made explicit which one you are talking about (for example in lines 423-425)

Reply:
Okay I went through and tried to clarify this section and the language used. Let me know if there are still outstanding questions!